### DISCUSSION TITLE
Votación extensión de tiempo de entrega de tareas

### DISCUSSION THREAD

**Discussion by joserasilla-uneat** [mmasias/progra2-22-23#107] [Polls]
Se ha planteado una extensión de los plazos de entrega de las tareas. Es necesario que todos los alumnos de la clase voten.
Actualmente se dan los enunciados el Viernes y el plazo de entrega es el domingo a las 23:59 horas.
Se propone la fecha de entrega el Jueves a las 23:59. Es decir, pasar de un fin de semana a 6 días.

**Comment by CarlosCasadoo** [comment]
![image](https://user-images.githubusercontent.com/55800474/223409693-47d4f3d3-43b8-40e0-93b3-cdd6425983ee.png)

**Comment by JoseSalceda** [comment]
![Screenshot_20230228-012548_WhatsApp.png](https://user-images.githubusercontent.com/114105383/223410210-9f0efffe-e271-4b1e-a04b-9e7691de218b.png)

**Reply by Xm3Ga** [reply]
padre

**Comment by TheMoys** [comment]
![imagen_2023-03-07_123357732](https://user-images.githubusercontent.com/52762460/223410640-2744aa98-136d-4ddc-bf60-4f99811f0197.png)

**Comment by Xm3Ga** [comment]
![20230227_121455](https://user-images.githubusercontent.com/77532409/223411265-3e6b7624-2cd5-4000-9b62-d3babf2a0232.jpg)
Que movida eono
